medusa/process_tv.py

```diff
@@ -489,6 +489,29 @@ def unrar(self, path, rar_files, force=False):
 
         return unpacked_files
 
+    def manually_searched(self, video_file):
+        """
+        Check the last occurrence for this release name in the history table is a manual searched.
+
+        :param video_file: File name
+        :return:
+        """
+        main_db_con = db.DBConnection()
+        history_result = main_db_con.select(
+            'SELECT resource LIKE ? '
+            'ORDER BY date DESC LIMIT 1',
+            ['%' + video_file])
+
+        if all(
+            [history_result,
+             history_result[0]['manually_searched'] == '1',
+             history_result[0]['action'] in (SNATCHED, SNATCHED_BEST, SNATCHED_PROPER)]
+        ):
+            self.log('Last snatch for this video was manual searched: {0}'.format(
+                video_file
+            ), logger.DEBUG)
+            return True
```

Comment:
The problem with that is that the `video_file` name doesn't always match the torrent/nzb name. So it's not 100% reliable.